playbook/app/pb_kits/playbook/pb_advanced_table/_advanced_table.tsx

```diff
@@ -234,9 +237,19 @@ const AdvancedTable = (props: AdvancedTableProps) => {
     },
 } : {}
 
-//initialize table
+  // Simulating chunked data fetch
+  const fetchSize = 20; // Number of rows per "page"
+  const [fullData] = useState(tableData); // All data from the JSON file
+  const [dataChunk, setDataChunk] = useState(fullData.slice(0, fetchSize)); // Initial chunk
+
+  // Virtualized table state
+  const [totalDBRowCount] = useState(fullData.length); // Total number of rows
+  const [totalFetched, setTotalFetched] = useState(fetchSize); // Track loaded rows
+  const [isFetching, setIsFetching] = useState(false);
+
+  //initialize table
   const table = useReactTable({
-    data: loading ? Array(loadingStateRowCount).fill({}) : tableData,
+    data: loading ? Array(loadingStateRowCount).fill({}) : dataChunk,
```

Comment:
Looks like we're changing the default of what we are passing to tanstack for data here by replacing tableData with dataChunk. I think we should have a conditional here, so if virtualizer prop exists, pass in dataChunk otherwise leave tableData. 

The dataChunk seems to be only giving 20 rows, and it looks fine in our docs since we don't have an example with more than 20 rows, but it may cause issues if devs in Nitro want a table with more than 20 rows and not use the virtual rows logic.